logdetective/server/server.py

```diff
@@ -420,6 +458,17 @@ async def perform_staged_analysis(
     )
 
 
+@app.get("/queue/print")
+async def queue_print(msg: str):
+    LOG.info("Will print %s", msg)
+    result = await enqueue_func(app.llm_queue, async_log, msg)
+    LOG.info("Printed %s and returned it", result)
```

Comment:
this looks very useful for monitoring

though shouldn't we return something?

Reply:
This wasn't really intended to stay in the code, it was just the equivalent of "printf debugging" to ensure that the enqueued function gets called with appropriate delays.